src/Tools/DynamoInstallDetective/ProductLookUp.cs

```diff
@@ -109,11 +109,11 @@ public interface IProductCollection
         IInstalledProduct GetLatestProduct();
 
         /// <summary>
-        /// Returns all installed product on the system using the given lookUp 
-        /// and initializes itself.
+        /// Get all installed products on the system using the given lookUp 
+        /// and append to Products property.
         /// </summary>
         /// <param name="lookUp">LookUp interface</param>
-        void LookUpAndInitProducts(IProductLookUp lookUp);
+        void LookUpAndAddProducts(IProductLookUp lookUp);
```

Comment:
to make me feel better can we mark this private?

Reply:
This need to stay as it is in interface